packages/web3/test/integration/external-providers/helper.ts

```diff
@@ -44,5 +46,31 @@ export async function performBasicRpcCalls(provider: SupportedProviders) {
 	expect(typeof blockNumber1).toBe('bigint');
 
 	// After sending a transaction, the blocknumber is supposed to be greater than or equal the block number before sending the transaction
-	expect(blockNumber1).toBeGreaterThanOrEqual(blockNumber0);
+	expect(blockNumber1).toBeGreaterThanOrEqual(blockNumber0);	
```

Comment:
Could you please ensure prettier is working well on your machine? Things link this should be fixed by it. 
```suggestion
	expect(blockNumber1).toBeGreaterThanOrEqual(blockNumber0);
```

Reply:
thank you, updated